webpack.main.js

```diff
@@ -28,7 +28,9 @@ const common = {
     output: {
         filename: '[name].js',
         path: path.resolve(__dirname, 'build'),
+        devtoolModuleFilenameTemplate: '[absolute-resource-path]',
     },
+    devtool: 'cheap-source-map'
```

Comment:
I don't think we'll want this devtools in production? We should either switch to a production ready one or toggle on/off depending on mode. https://webpack.js.org/configuration/devtool/

Reply:
Good point. I'm thinking `source-map` since it matches our other clients. However, I'm not sure if these source maps are useful in production at all.

Does anyone know if you can attach to a production instance of an electron app's main process?

Reply:
https://stackoverflow.com/questions/45485262/how-to-debug-electron-production-binaries seems to be possible.

Reply:
Per our discussion I went with building source maps only in dev builds